src/EnergyPlus/Autosizing/Base.cc

```diff
@@ -230,7 +230,7 @@ void BaseSizer::preSize(EnergyPlusData &state, Real64 const _originalValue)
 
     if (this->curSysNum > 0 && this->curSysNum <= this->numPrimaryAirSys) {
         if (this->sysSizingRunDone) {
-            for (auto &sizingInput : this->sysSizingInputData) {
+            for (auto const &sizingInput : this->sysSizingInputData) {
```

Comment:
What is the benefit of using const?

Reply:
Two benefits. First, it potentially allows the compiler to perform some additional optimizations that rely on the fact that the object pointed to by `sizingInput` will not change within the scope. Second, it documents the fact that the object will not change during the scope.